apps/prime/tsconfig.app.json

```diff
@@ -9,6 +9,9 @@
       "vite/client"
     ]
   },
+  "files": [
+    "../../libs/shared/theme/src/theme.d.ts"
```

Comment:
this dApp doesn't use this theme, I don't think this is necessary

Reply:
I had to build a shoe due to the requirechain component

Reply:
Until now nothing inside of shared has used `variant={'action'}` but RequireChain does.

Reply:
I've created `../../libs/prime/theme/src/theme.d.ts` now and added the `action` variant to it. 🧐

Reply:
nop, that solution does not scale nor is error proff, you cannot presume a custom variant will be present in the theme of the app that will consume your component. In other words, you cannot use a custom variant in a generic component, those are 2 different scopes.
There are 2 way of solving this:
- either you only use default theme attributes in your generic component and you derive custom styles
- either you expose the button props through your parent component props
I tend to try to keep it simple and go for the first

Reply:
Or even easier solution is to completely drop `RequireChain` component as it's not used anywhere 😝

Reply:
I'm okay with either. I think it's relatively unlikely we are ever going to want to use this. I found it to be an overall bad experience.

Reply:
Ok, let's revert it all then